services/user_service/database_connector.py

```diff
@@ -0,0 +1,64 @@
+import json
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db
+
+#implements CRUD operations interface
```

Comment:
![image](https://user-images.githubusercontent.com/32463897/141324434-c014a690-2a45-4999-9335-f8866959bb3a.png)

It is the only your comment in PR. I think we can remove it without loosing anything